### PR TITLE
feat: update README

### PR DESCRIPTION
**Pull request by artemrys**
This will the first round of the README updates.

### CODE REVIEW

File: README-addons.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Documentation on how to build / test add-ons
+
+## Build
```
Comment (uoboda-splunk) [inline README-addons.md:3]: Maybe?
```suggestion
## Local build
```

File: README-addons.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+All the add-ons are using [`poetry`](https://github.com/python-poetry/poetry) as a package management tool.
```
Comment (uoboda-splunk) [inline README-addons.md:5]: Maybe add info about Python version?

File: tools/sync.sh [hunk 3]
```diff
@@ -271,6 +271,10 @@ else
         git rm "LICENSES/.DS_Store" || true
     fi
 
+    if [[ -f "README.md" ]]; then
```
Comment (uoboda-splunk) [inline tools/sync.sh:274]: We are updating it with enforce right? Why to rm it?

File: README-addons.md (hunk 2 above)
Comment (artemrys) [reply README-addons.md:5]: Done

**Pull request merged**